vendor/github.com/grafana/dskit/cache/lru.go

```diff
@@ -49,19 +47,19 @@ func WrapWithLRUCache(c Cache, name string, reg prometheus.Registerer, lruSize i
 		defaultTTL: defaultTTL,
 
 		requests: promauto.With(reg).NewCounter(prometheus.CounterOpts{
-			Name:        "cortex_cache_memory_requests_total",
+			Name:        "cache_memory_requests_total",
```

Comment:
This is a breaking change for Mimir. It's fine doing this change in dskit, but then in Mimir you should wrap the registerer to add the `cortex_` prefix back.